packages/block-editor/src/store/selectors.js

```diff
@@ -2287,6 +2287,10 @@ export const __experimentalGetParsedPattern = createRegistrySelector(
 					metadata: {
 						...( blocks[ 0 ].attributes.metadata || {} ),
 						categories: pattern.categories,
+						patternName: pattern.name,
```

Comment:
Nit, but I wonder if these could be grouped together:
```js
shuffle: {
    categories: pattern.categories,
    name: pattern.name,
}
```

Reply:
That does mean we can be consistent with naming which is nicer, but they may be used for things other than shuffle...

Reply:
True. The `name` could be useful for a few things.

The `categories` are only for the originally inserted pattern though, as a user shuffles, the newly shown pattern might have different categories to those stored in the attribute (though will always have one category in common), so that feels more like a search parameter for the shuffle feature.